src/Abp.Zero/Zero/Localization/Source/AbpZero.xml

```diff
@@ -1,39 +1,40 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Dear Halil, first of all thank you for a wonderful  framework.
Sorry about tabs. I committed additional changes. Except that AbpZero.xml also has changes on 2FA localization XML format.

Before:
`<text name="Sms"></text>`
Currently is:
`<text name="Sms" value="Sms" />`